src/nunit.analyzers.tests/DiagnosticSuppressors/DereferencePossiblyNullReferenceSuppressorTests.cs

```diff
@@ -240,50 +234,50 @@ public void Test([Values] bool create)
                 {ABDefinition}
```

Comment:
`↓` in line 231 - `Assert.That(↓a.B.Text.Length, Is.GreaterThan(0));`

Reply:
Done.